src/main/charts/bitbucket/values.yaml

```diff
@@ -35,6 +35,11 @@
 #
 replicaCount: 1
 
+# -- The termination grace period for pods during shutdown. This
+# should be set the Bitbucket internal grace period (default 30
```

Comment:
Not sure if this sentence reads correctly: `should be set the Bitbucket internal grace period ` should it be `should will set the Bitbucket internal grace period ` ? 

This needs to be fixed for the other products too.

Reply:
Bitbucket has it's own internal grace-period for SSH etc, which the customer can override. This setting is for the Kubernetes grace setting, which is separate. The Kubernetes one should be slightly higher than the BB one, so that BB can complete it's shutdown before the Kubernetes grace period times out.

But there is a missing word, I'll fix that.